src/styles/theme/themes/dark.ts

```diff
@@ -46,6 +46,7 @@ const darkTheme = {
     shadow: colors.black,
     componentBG: colors.productDark100,
     hoverComponentBG: colors.productDark300,
+    messageHighlight: colors.messageHightlightDark,
```

Comment:
```suggestion
    messageHighlightBG: colors.messageHightlightDark,
```
Do you think `messageHighlightBG` will be a good fit here?

Reply:
Sound good.